test/jdk/java/awt/dnd/InterJVMGetDropSuccessTest/InterJVMGetDropSuccessTest.java

```diff
@@ -29,16 +29,37 @@
   @run main InterJVMGetDropSuccessTest
 */
 
-import java.awt.*;
-import java.awt.datatransfer.*;
-import java.awt.dnd.*;
-import java.awt.event.*;
-import java.io.*;
+import java.awt.Frame;
+import java.awt.Component;
+import java.awt.Point;
+import java.awt.Dimension;
+import java.awt.EventQueue;
+import java.awt.Toolkit;
+import java.awt.AWTEvent;
+import java.awt.Robot;
+import java.awt.event.AWTEventListener;
+import java.awt.event.InputEvent;
+import java.awt.event.MouseEvent;
+import java.awt.datatransfer.StringSelection;
+import java.awt.datatransfer.Transferable;
+import java.awt.dnd.DropTarget;
+import java.awt.dnd.DragGestureListener;
+import java.awt.dnd.DragSourceAdapter;
+import java.awt.dnd.DragSourceDropEvent;
+import java.awt.dnd.DnDConstants;
+import java.awt.dnd.DropTargetDropEvent;
+import java.awt.dnd.DropTargetAdapter;
+import java.awt.dnd.DragGestureRecognizer;
+import java.awt.dnd.DragSource;
+import java.awt.dnd.DropTargetListener;
+import java.awt.dnd.DragGestureEvent;
+import java.io.File;
+import java.io.InputStream;
 
 public class InterJVMGetDropSuccessTest {
 
     private int returnCode = Util.CODE_NOT_RETURNED;
-    private boolean successCodes[] = { true, false };
+    private final boolean successCodes[] = new boolean[]{ true, false };
```

Comment:
```suggestion
    private final boolean[] successCodes = { true, false };
```
Use Java-style array declaration; it means the brackets are with the type `boolean[]` rather than the field/variable name.